.github/workflows/assign-pr.yml

```
@@ -8,7 +8,7 @@ jobs:
   assign_creator:
     runs-on: ubuntu-latest
```

The job should have explicit permissions to limit the scope of the token:
```suggestion
    runs-on: ubuntu-latest
    permissions:
      pull-requests: write
```